ansible.go

```diff
@@ -139,6 +144,101 @@ type AnsiblePlaybookRunner struct {
 	Env             []string // Envvars to pass into the Ansible run
 }
 
+// Parse Ansible Run Status from output
+func parseAnsibleRunStats(ansibleOutput AnsibleRunOutput) (AnsibleRunOutput, error) {
+	var err error
+
+	if ansibleOutput.CommandOutput.Error != nil {
```

Comment:
nit: This is not a parse error. Let's take it out of `parseAnsibleRunStats` and put it back to `Run` function right before calling `parseAnsibleRunStats`.